pallets/fragments/src/lib.rs

```diff
@@ -759,12 +759,12 @@ pub mod pallet {
 				)
 				.map_err(|_| Error::<T>::InsufficientBalance)?;
 			} else {
-				<pallet_balances::Pallet<T> as Currency<T::AccountId>>::transfer(
+				<pallet_balances::Pallet<T> as fungible::Transfer<T::AccountId>>::transfer(
```

Comment:
confused on this, balances does not implement fungibles afaik? if it does we need to prevent transfer there too

Reply:
It is blocked now on Substrate. In the last two commits [here](https://github.com/fragcolor-xyz/clamor/pull/166/commits/a2098fc6b36d3157dd012e1b0b1dd8f2c54ecb34) and [here](https://github.com/fragcolor-xyz/clamor/pull/166/commits/a380f993efb11a3d60eb17f65ee747878f0825db) I call `do_transfer` from `pallet_balances`, which is now public and I also removed the use of `fungible` trait here.